View3D/Components/Gizmo/GizmoComponent.cs

```diff
@@ -117,8 +115,17 @@ private void GizmoScaleEvent(ITransformable transformable, TransformationEventAr
 
         public override void Update(GameTime gameTime)
         {
-            if (!(_selectionManager.GetState().Mode == GeometrySelectionMode.Object || _selectionManager.GetState().Mode == GeometrySelectionMode.Vertex))
-                return;
+            var selectionMode = _selectionManager.GetState().Mode;
+            switch (selectionMode)
```

Comment:
What does this switch do? 
Seems like nothing, but I might be missing something?